ecdsa/party/implementation.go

```diff
@@ -375,3 +397,13 @@ func (p *Impl) reportError(newError *tss.Error) {
 		// no one is waiting on error reporting channel/ no buffer.
 	}
 }
+
+func (p *Impl) selfInSigningCommittee(parties []*tss.PartyID) *tss.PartyID {
+	for _, party := range parties {
+		if party.Id == p.partyID.Id && bytes.Equal(party.Key, p.partyID.Key) && party.Moniker == p.partyID.Moniker {
```

Comment:
should we check moniker equality?

Reply:
I don't follow. 
Do you ask whether we should consider it?  Or whether we check it? 

In the case of the latter, we check it. 
In the case of the former, it doesn't hurt, though indeed, we don't have to.

Reply:
Right, I see that we check it. I'm not sure if we should, since it's just a nickname

Reply:
I'll remove it.